app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui compact stackable grid">
```

Comment:
@anu-007 Please use https://chrome.google.com/webstore/detail/web-developer/bfbameneiokkgbdmiekhjnmfkcnldhhm for checking responsive layouts.